db/schema.rb

```diff
@@ -159,6 +159,7 @@
     t.text     "credits"
     t.decimal  "height",             :precision => 8, :scale => 2
     t.string   "orientation"
+    t.string   "subtype"    
```

Comment:
Considering `null: false`. Also, consider using [Postgres Enumerated Types](http://www.postgresql.org/docs/current/static/datatype-enum.html) to limit what this column can contain, which is more beneficial.